infrastructure/openshift/retrieve-pnet-logs.sh

```diff
@@ -7,10 +7,14 @@ main() {
 
   echo -e "-------- STARTING CRON --------\n"
 
-  LAST_TX_ID=$(psql -h ${PGHOST} -d ${PGDATABASE} -U ${PGUSER} -W ${PGPASSWORD} -c 'select max(ptl."TransactionId") from "PharmanetTransactionLog" ptl')
+#  LAST_TX_ID=$(psql -h ${PGHOST} -d ${PGDATABASE} -U ${PGUSER} -W ${PGPASSWORD} -c 'select max(ptl."TransactionId") from "PharmanetTransactionLog" ptl')
+  LAST_TX_ID=$(psql -h ${PGHOST} -d ${PGDATABASE} -U ${PGUSER} -W ${PGPASSWORD} -c 'select count(*) from "Enrollee" e')
   echo -e ${LAST_TX_ID}
 
   ls -l /opt/certs
+
+  curl --cert /opt/certs/prime-odr-api-cert.crt:pwd4primetst --key /opt/certs/prime-odr-api-cert.key --header 'Authorization: Basic cHJpbWVkYXRhdXNlcjpWaWM5I1RzdDE=' -X GET https://t1primedatasvc.maximusbc.ca/odr/prime/pnetdata/transactionLog
```

Comment:
I see two secrets in this line of code. They are now compromised and we need to ask HIBC to rotate them.

Reply:
HIBC has completed updating 2021-09-08 and we are now using new secrets

Reply:
@itchison Please resolve conversation if appropriate